generate-stackbrew-library.sh

```diff
@@ -4,7 +4,8 @@ set -e
 declare -A aliases
 aliases=(
 	[2.8]='2'
-	[3.0]='3 latest'
+	[3.0]=''
```

Comment:
I think we should just remove this line altogether (since we don't need any aliases for 3.0 anymore now that 3.2 is GA).